CREATE TABLE LIKE copies hidden columns

ericharmeling opened the pull request:
Fixes https://github.com/cockroachdb/docs/issues/11380.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/docs/12255)
<!-- Reviewable:end -->

stbof commented on v21.2/create-table.md:
```diff
@@ -127,9 +127,9 @@ The following options are supported:
-Column families, partitioning, interleavings, and foreign key constraints
-cannot be preserved from the old table and will have to be recreated
-manually in the new table if the user wishes.
+`CREATE TABLE LIKE` statements cannot copy [column families](column-families.html), [partitions](partitioning.html), and [foreign key constraints](foreign-key.html) from old tables. If you want these column qualifications in a new table, you must recreate them manually.
```
```suggestion
`CREATE TABLE LIKE` statements cannot copy [column families](column-families.html), [partitions](partitioning.html), and [foreign key constraints](foreign-key.html) from other tables. If you want these column qualifications in a new table, you must recreate them manually.
```

ericharmeling replied:
Done

stbof commented on v21.2/create-table.md:
```diff
@@ -127,9 +127,9 @@ The following options are supported:
-manually in the new table if the user wishes.
+`CREATE TABLE LIKE` statements cannot copy [column families](column-families.html), [partitions](partitioning.html), and [foreign key constraints](foreign-key.html) from old tables. If you want these column qualifications in a new table, you must recreate them manually.
+
+`CREATE TABLE LIKE` copies all hidden columns (e.g., the hidden [`crdb_region`](set-locality.html#crdb_region) column in multi-region databases) from old tables to new tables.
```
```suggestion
`CREATE TABLE LIKE` copies all hidden columns (e.g., the hidden [`crdb_region`](set-locality.html#crdb_region) column in multi-region databases) from the existing table to the new table.
```

ericharmeling replied:
Done

stbof reviewed (approved):
I find the word `old` to be kind of strange. My suggested replacement is `existing`.

ericharmeling reviewed (commented):
TFTR!

ericharmeling commented:
> I find the word `old` to be kind of strange. My suggested replacement is `existing`.

I agree. I updated it here. Unfortunately, I think we use this kind of terminology in a lot of docs where we could use "existing" instead. I'll try to update it where I see it.

In general, using "old" could be perceived as non-inclusive (in addition to not being the most precise term), so I also added it to the ["Write about features and users in inclusive ways" section of the style guide](https://github.com/cockroachdb/docs/wiki/Style-Guide#write-about-features-and-users-in-inclusive-ways).
